doc/rados/operations/devices.rst

```diff
@@ -25,6 +25,22 @@ location and how it is being consumed with::
 
   ceph device info <devid>
 
+Identifying physical devices
+----------------------------
+
+You can blink lights on hardware enclosures to make the replacement of failed
+disks easy and less error-prone.  Use the following command::
+
+  device light on|off <devid> [ident|fault] [--force]
+
+The <devid> parameter is the device identification, and you can obtain this
+information using the command::
+
+  ceph device ls
+
+The second parameter sets the kind of light to blink.  By default the
```

Comment:
```suggestion
The second parameter of ``device light on|off`` command sets the kind of light to blink.  By default the
```

as otherwise, reader (me, for instance) might want to search backwards for the command and the "second parameter". and the first match would be "ceph device ls".

Reply:
Changed. @tchaikov,  what do you think now?